betelgeuse.py

```diff
@@ -237,6 +237,7 @@ def add_test_case(args):
         # automated, manualonly, and notautomated
         auto_status = 'automated' if test.automated else 'notautomated'
         caseposneg = 'negative' if 'negative' in test.name else 'positive'
+        casecomponent = test.case_component if test.case_component else '-'
```

Comment:
@Akasurde as I said testimony will not be handling all Polarion fields that should be used the unexpected tags instead. In any case I will wait for a position from @sthirugn on your PR there.